controllers/auth.js

```diff
@@ -0,0 +1,55 @@
+//inport model
+const User = require('../models/User')
+const { StatusCodes } = require('http-status-codes')
+const { BadRequestError , UnauthenticatedError} = require('../errors')
+
+
+const register = async (req, res) => {
+     
+        //create User 
+        const user = await User.create({ ...req.body })
```

Comment:
One thing to consider for a production app is making sure the req.body is in the form/has the fields you’d expect. If you try this with a partially formed body, or one that has too many fields, could still go through